frontend/src/metabase/common/components/DatePicker/RelativeDatePicker/constants.ts

```diff
@@ -0,0 +1,22 @@
+import { t } from "ttag";
+import type { DatePickerTruncationUnit } from "../types";
+import type { Tab } from "./types";
+
+export const TABS: Tab[] = [
+  { label: t`Past`, direction: "last" },
+  { label: t`Current`, direction: "current" },
+  { label: t`Next`, direction: "next" },
+];
+
+export const UNIT_GROUPS: DatePickerTruncationUnit[][] = [
+  ["day", "week", "month"],
+  ["quarter", "year"],
+];
```

Comment:
This one doesn't seem to be used anywhere. `CurrentDatePicker` has its own definition of `UNIT_GROUPS` (in the constants) file